pkg/parsers/kernel/kernel.go

```diff
@@ -8,20 +8,21 @@ import (
 	"fmt"
 )
 
-type KernelVersionInfo struct {
+// VersionInfo holds informations about the kernel
+type VersionInfo struct {
 	Kernel int
 	Major  int
 	Minor  int
 	Flavor string
```

Comment:
hum ``golint`` did not say a thing about that, but I'll document them anyway :wink:.